### PR TITLE
chore: fix stylus loader link

### PR DESCRIPTION
**Pull request by anshumanv**
Missed in https://github.com/webpack/webpack.js.org/pull/3903

- [x] Read and sign the [CLA][1]. PRs that haven't signed it won't be accepted.
- [x] Make sure your PR complies with the [writer's guide][2].
- [x] Review the diff carefully as sometimes this can reveal issues.
- [x] Do not abandon your Pull Request: [Stale Pull Requests][3].
- __Remove these instructions from your PR as they are for your eyes only.__

[1]: https://cla.js.foundation/webpack/webpack.js.org
[2]: https://webpack.js.org/contribute/writers-guide/
[3]: https://webpack.js.org/contribute/#pull-requests

### CODE REVIEW

**Review by EugeneHlushko — APPROVED** [review]

File: src/content/loaders/index.md [hunk 1]
```diff
@@ -62,7 +62,7 @@ Loaders are activated by using `loadername!` prefixes in `require()` statements,
 - [`less-loader`](/loaders/less-loader) Loads and compiles a LESS file
 - [`sass-loader`](/loaders/sass-loader) Loads and compiles a SASS/SCSS file
 - [`postcss-loader`](/loaders/postcss-loader) Loads and transforms a CSS/SSS file using [PostCSS](http://postcss.org)
-- [`stylus-loader`](https://github.com/shama/stylus-loader) Loads and compiles a Stylus file
+- [`stylus-loader`](https://github.com/webpack-contrib/stylus-loader) Loads and compiles a Stylus file
```
Comment (EugeneHlushko) [inline src/content/loaders/index.md:65]:
```suggestion
- [`stylus-loader`](/loaders/stylus-loader/) Loads and compiles a Stylus file
```

**Review by EugeneHlushko — CHANGES_REQUESTED** [review]
i think its now available through the docs

File: src/content/loaders/index.md (hunk 1 above)
Comment (anshumanv) [reply src/content/loaders/index.md]: Done!

**Review by EugeneHlushko — APPROVED** [review]

**Comment by EugeneHlushko** [comment]
Maybe the branch is outdated, stylus page wasnt available on preview. Trying with update now

**Comment by EugeneHlushko** [comment]
Thanks!

**Pull request merged**